Source/Components/LoyaltyHeader/MLBusinessLoyaltyHeaderView.swift

```diff
@@ -60,10 +62,22 @@ private extension MLBusinessLoyaltyHeaderView {
         self.ringView?.innerRingColor = ringHexaColor.hexaToUIColor()
         self.ringView?.outerRingColor = secondaryHexaColor.hexaToUIColor()
         self.ringView?.innerCenterText = String(ringNumber)
+        self.ringView?.isHidden = false
         
         if self.fillPercentProgress {
             self.ringView?.startProgress(to: CGFloat(ringPercentage), duration: 0)
         }
+        
+        if(viewData?.getSubtitle() != nil){
+            if(viewData?.getRingNumber() == nil){
+                if(self.ringView != nil){
+                    hideRing(self.ringView)
+                }
+            }
+            if(viewData?.getTitle() == nil){
```

Comment:
No hacen falta los parentesis
```suggestion
            if viewData?.getTitle() == nil {
```